File: 1des/fpoo/aula03/ex1.c
/*
	Desenvolva um programa que leia três
	variáveis (a, b, c) e resolva a
	expressão: ( a + b ) / c.
*/
#include<stdio.h>
#include<locale.h>
int main(){
	setlocale(LC_ALL,""); //Configura o programa para aceitar acentos
	//Configuração das variáveis
	int a, b, c, d; //Este programa utiliza 128 bits 16 bytes

	//Entradas do programa
	printf("Digite um valor para a = ");
	scanf("%d", &a);
	printf("Digite um valor para b = ");
	scanf("%d", &b);
	printf("Digite um valor para c = ");
	scanf("%d", &c);

	//Processamento
	d = (a + b) / c;
	
	//Saída do programa
	printf("O resultado da conta (a + b) / c = %d",d);
}

File: old/1des/fpoo/workspace/CorrecaoLista1e2/src/lista1/Ex1.java
package lista1;
import java.util.Scanner;

/* Desenvolva um programa que leia três variáveis (a, b, c)
 * e resolva a expressão: ( a + b ) / c */
public class Ex1 {

	public static void main(String[] args) {
		Scanner leia = new Scanner(System.in);
		System.out.println("Digite valores inteiros para A, B, C:");
		int a = leia.nextInt();
		int b = leia.nextInt();
		int c = leia.nextInt();
		float resultado = (float)(a + b) / c;
		System.out.printf("O resultado da (A + B) / C = %.2f",resultado);
	}

}
